src/Apps/Components/AppToasts.tsx

```diff
@@ -1,27 +1,15 @@
 import * as React from "react"
 import { Box, Column, GridColumns, Toasts, useToasts } from "@artsy/palette"
-import { useNavBarHeight } from "Components/NavBar/useNavBarHeight"
 import { AppContainer } from "./AppContainer"
 import { Z } from "./constants"
 
-interface AppToastsProps {
-  accomodateNav: boolean
-}
-
-export const AppToasts: React.FC<AppToastsProps> = ({ accomodateNav }) => {
-  const { height } = useNavBarHeight()
+export const AppToasts: React.FC = () => {
   const { toasts } = useToasts()
 
   if (toasts.length === 0) return null
 
   return (
-    <Box
-      position="fixed"
-      zIndex={Z.toasts}
-      top={accomodateNav ? height : 0}
-      left={0}
-      width="100%"
-    >
+    <Box position="fixed" zIndex={Z.toasts} bottom={4} left={0} width="100%">
```

Comment:
4 plus the 1 below equals an invalid value. Should just use the py below to vertically place

Reply:
oh, gotcha